### PR TITLE
fix: set latest as the default if no tag or id is provided on addi…

### LINKED ISSUE #154: Translate AdditionalImages missing tags as latest

### PR DESCRIPTION
**Pull request by jpower432**
…tional images

Fixes #154

# Description

This PR will assume the tag or ID component is none is provided with `AdditionalImage` spec. Tag will be set as latest.

Fixes #154 

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] Unit test added to test additional images without tags or digest

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: pkg/cli/mirror/additional_test.go [hunk 1]
```diff
@@ -42,12 +30,82 @@ func Test_GetAdditional(t *testing.T) {
...
+			},
+			want:    ErrBlocked{},
+			wantErr: true,
+		},
+	}
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+
+			assocs, err := opts.GetAdditional(test.cfg, test.cfg.Mirror.AdditionalImages)
+			t.Log(err)
```
Comment (estroz) [inline pkg/cli/mirror/additional_test.go:93]: Remove these

Comment (jpower432) [reply pkg/cli/mirror/additional_test.go:93]: Thank you! Mistakenly left that in.

**Review by estroz — COMMENTED** [review]
/lgtm

**Pull request merged**